code/_onclick/MouseDrag.dm

```diff
@@ -13,11 +19,21 @@
 		gun.set_autofire(over_object, src)
 
 /mob/proc/OnMouseDown(atom/object, location, control, params)
+	if(istype(loc, /atom))
```

Comment:
These checks could just be `if(loc)`, loc being a non-null non-atom is impossible as far as I am aware.